abel/transform.py

```diff
@@ -28,19 +29,38 @@ class Transform(object):
   
     Attributes
     ----------
-    transform : numpy 2D array 
+    transform: numpy 2D array 
```

Comment:
I think that the extra space before the colon is the numpy-style docstring standard: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html